cmd/test/main_test.go

```diff
@@ -814,15 +814,15 @@ func executeTestCommand(ctx *TestCommandContext, args []string) error {
 
 func test_testCommand_output_flags_validation(t *testing.T, ctx *TestCommandContext) {
 
-	validOutputValues := [5]string{"simple", "json", "yaml", "xml", "JUnit"}
+	validOutputValues := [...]string{"", "simple", "json", "yaml", "xml", "JUnit"}
```

Comment:
[...]


![Alt Text](https://media1.giphy.com/media/l4q8cJzGdR9J8w3hS/giphy.gif)

Reply:
`"" ` - is for empty case right?

Reply:
yep!